tests/playwright/src/model/pages/create-kind-cluster-page.ts

```diff
@@ -120,12 +120,16 @@ export class CreateKindClusterPage extends BasePage {
 
   private async createCluster(timeout: number = 300_000): Promise<void> {
     return test.step('Create cluster', async () => {
-      await playExpect(this.clusterCreationButton).toBeVisible();
-      await this.clusterCreationButton.click();
-      await this.logsButton.scrollIntoViewIfNeeded();
-      await this.logsButton.click();
-      await playExpect(this.goBackButton).toBeVisible({ timeout: timeout });
-      await this.goBackButton.click();
+      try {
+        await playExpect(this.clusterCreationButton).toBeVisible();
+        await this.clusterCreationButton.click();
+        await this.logsButton.scrollIntoViewIfNeeded();
+        await this.logsButton.click();
+        await playExpect(this.goBackButton).toBeVisible({ timeout: timeout });
+        await this.goBackButton.click();
+      } catch (error) {
+        throw Error(`${await this.errorMessage.textContent()}`);
+      }
```

Comment:
I would refactor the code to something more error-sensible.
Here basically any error can result in situation where the errorMessage will not be present and still will throw an error. Ideal would be that we will end up with assertionError since we did not manage to start up cluster.
I would use combination of soft asserts and count method for locators. Something like:
```
        await playExpect.soft(this.clusterCreationButton).toBeEnabled();
        await this.clusterCreationButton.click();
        await this.logsButton.scrollIntoViewIfNeeded();
        await playExpect.soft(this.logsButton).toBeEnabled();
        await this.logsButton.click();
        let errorText = '';
        if ((await this.errorMessage.count()) > 0) {
          errorText = await this.errorMessage.textContent();
        }
        await playExpect(errorMessage, `Failed to create kind cluster: ${errorText}`).not.toBeVisible();
        await playExpect(this.goBackButton).toBeEnabled({ timeout: timeout });
        await this.goBackButton.click();
```